int32 QUERY=0          # State constant - query
int32 SEARCH=1         # State constant - searching for balls
int32 TURN=2           # State constant - turn robot around
int32 MOVETOBALL=3     # State constant - move towards ball
int32 PICKBALL=4       # State constant - pick the ball
int32 MOVETOSTORAGE=5  # State constant - move the ball towards the storage
int32 DROPBALL=6       # State constant - drop the ball
int32 flowid
